lib/beaker-hostgenerator/hypervisor/docker.rb

```diff
@@ -17,7 +17,7 @@ def generate_node(node_info, base_config, bhg_version)
         end
 
         base_config['image'] = image(ostype)
-        base_config['image'].prepend('amd64/') if node_info['bits'] == '64' && !base_config['image'].start_with?('quay.io/')
+        base_config['image'].prepend('amd64/') if node_info['bits'] == '64' && base_config['image'] !~ /^(quay\.io|registry\.access\.redhat\.com)/
```

Comment:
You can give multiple prefixes to `start_with?` so I'd expect this to work:
```suggestion
        base_config['image'].prepend('amd64/') if node_info['bits'] == '64' && !base_config['image'].start_with('quay.io', 'registry.access.redhat.com')
```